components/CreateForm.js

```diff
@@ -60,7 +121,30 @@ function CreateForm() {
       </FormContainer>
       <ButtonContainer>
         <Button ref={buttonRef} isPrimary onClick={handleAddStep}>
-          Add step
+          <IconContext.Provider
+            value={{
+              color: "inherit",
+              size: "1.4em",
+              title: "checkmark icon",
+              style: { marginRight: "0.5em" },
+            }}
+          >
+            <TbPlus />
```

Comment:
For a single item you can leave the IconContext away and just give the item the props directly. This also makes it more visible that you are not using styled components here.